plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -45,6 +45,7 @@ public class IcebergTableProperties
     public static final String FORMAT_VERSION_PROPERTY = "format_version";
     public static final String ORC_BLOOM_FILTER_COLUMNS_PROPERTY = "orc_bloom_filter_columns";
     public static final String ORC_BLOOM_FILTER_FPP_PROPERTY = "orc_bloom_filter_fpp";
+    public static final String PARQUET_BLOOM_FILTER_COLUMNS_PROPERTY = "parquet_bloom_filter_columns";
```

Comment:
```
spark-sql (default)> CREATE TABLE t1 (testInteger INTEGER, testLong BIGINT, testString STRING, testDouble DOUBLE, testFloat REAL)
                   > USING iceberg
                   > TBLPROPERTIES (
                   > 'write.parquet.bloom-filter-enabled.column.testInteger' = true, 
                   > 'write.parquet.bloom-filter-enabled.column.testLong' = true,
                   > 'write.parquet.bloom-filter-enabled.column.testString' = true,
                   > 'write.parquet.bloom-filter-enabled.column.testDouble' = true,
                   > 'write.parquet.bloom-filter-enabled.column.testFloat' = true
                   > );
```

```
trino> show create table iceberg.default.t1;
                           Create Table                           
------------------------------------------------------------------
 CREATE TABLE iceberg.default.t1 (                                
    testinteger integer,                                          
    testlong bigint,                                              
    teststring varchar,                                           
    testdouble double,                                            
    testfloat real                                                
 )                                                                
 WITH (                                                           
    format = 'PARQUET',                                           
    format_version = 2,                                           
    location = 'hdfs://hadoop-master:9000/user/hive/warehouse/t1' 
 )                                                                
(1 row)
```

Shouldn't we see in `SHOW CREATE TABLE` the bloom filter columns now that we're dealing with a supported table property?

Reply:
Modify `io.trino.plugin.iceberg.IcebergUtil#getIcebergTableProperties`

Reply:
i tried on the above scaffolding 

```
SELECT COUNT(*) FROM iceberg.default.t1 where testInteger in (9444, -88777, 6711111);
```

and see the following

```
"queryStats" : {
  ....
    "physicalInputDataSize" : "656400B",
    "failedPhysicalInputDataSize" : "0B",
    "physicalInputPositions" : 5,
```

This seems not to overlap with the expectations from `io.trino.testing.BaseTestParquetWithBloomFilters#testBloomFilterRowGroupPruning(io.trino.spi.connector.CatalogSchemaTableName, java.lang.String)`

Reply:
We could add a `toLowerCase` to `getParquetBloomFilterColumns` to handle this? It looks like we have the same issues for the Iceberg ORC Bloom filters. Should we handle case sensitivity in this PR, or handle it in a follow up?

Reply:
Is it possible to access the `queryStats` above from a product test?

Reply:
Well, if you're into some late Friday hacking :) you could try

https://github.com/trinodb/trino/blob/ca209630136eabda2449594ef2b6a4d82fb9c2e5/testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveConnectorKerberosSmokeTest.java#L88-L90

which will get you the `queryId`

With the query http://trino:8080/ui/api/query/20240503_152311_00002_f7nv5 you should be getting the JSON containing the QueryInfo and from there you are getting the long awaited `queryStats`

Reply:
An alternative with less headaches would be to register a pre-created resource table and check the query stats on it similar to what has been done on https://github.com/trinodb/trino/blob/ca209630136eabda2449594ef2b6a4d82fb9c2e5/plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergReadVersionedTableByTemporal.java

Reply:
We need a mechanism to get the query stats in the product tests to ensure that the bloom filter is actually effective and we don't introduce while refactoring regressions.

Reply:
Would someone be able to help add this logic? I don't have much experience with the product tests and unfortunately don't have much capacity to follow up on this at the moment. It would be much appreciated!

Reply:
@findinpath aren't we already testing effectiveness of bloom filter in query runner tests ? I'm not sure that we should block this PR over checking this in product tests as well, we don't do that with Apache Hive for bloom filters in hive connector as well.